doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,83 @@
+
+ ``FLUSH INDEX_STATISTICS`` / ``FLUSH CHANGED_PAGE_BITMAPS`` and ``FLUSH USER_STATISTICS`` / ``RESET CHANGE_PAGE_BITMAPS`` pairs of commands were inadvertently joined, i.e. issuing either command had the effect of both. The first pair, besides flushing both index statistics and changed page bitmaps, had the effect of ``FLUSH INDEX_STATISTICS`` requiring ``SUPER`` instead of ``RELOAD`` privilege. The second pair resulted in ``FLUSH USER_STATISTICS`` destroying changed page bitmaps. Bug fixed :bug:`1472251`.
+
+ Enabling :variable:`super_read_only` together with :variable:`read_only` in :file:`my.cnf` would result in server crashing on startup. The workaround is to enable :variable:`super_read_only` dynamically on a running server.Bug fixed :bug:`1389935` ( the fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ Enabling :variable:`super_read_only` as a command line option would not enable :variable:`read_only`. Bug fixed :bug:`1389935` ( the fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ If a new connection thread was created while a ``SHOW PROCESSLIST`` command or a :table:`INFORMATION_SCHEMA.PROCESSLIST` query was in progress, it could have a negative TIME_MS value returned in the PROCESSLIST output. Bug fixed :bug:`1379582`.
+
+ Running ``INSERT`` queries for pessimistic B-tree inserts with :ref:`innodb_fake_changes_page` enabled could lead to server assertion with ``unknown error code 1000``. Bug fixed :bug:`1410410`.
+
+ Running ``ALTER TABLE ... DISCARD TABLESPACE`` with :ref:`innodb_fake_changes_page` enabled would lead to a server assertion. Bug fixed :bug:`1372219`.
+
+ ``ALTER TABLE`` did not allow to change a column to ``NOT NULL`` if the column was referenced in a foreign key. Bug fixed :bug:`1470677` (upstream :mysqlbug:`77591`).
+
+ ``DROP TABLE IF EXISTS`` which fails due to a foreign key presence could break replication if slave had replication filters. Bug fixed :bug:`1475107` (upstream :mysqlbug:`77684`).
+
+ Enabling :ref:`log_archiving` when :option:`--innodb-read-only` option was enabled would cause server to crash. Bug fixed :bug:`1484432`.
+
+ LRU manager thread flushing was being accounted to ``buffer_flush_background`` *InnoDB*  metrics which was wrong and redundant. Bug fixed :bug:`1491420`.
+
+ Fixed a typo in the cleaner thread loop where ``n_flushed`` is added to, instead of reset, by the idle server flushing. This may cause a cleaner thread sleep skip on a non-idle server. Bug fixed :bug:`1491435`.
+
+ Running |TokuDB| for a long time with lots of open/closing file could lead to a server crash due to server incorrectly setting a reserved value. Bug fixed :ftbug:`690`. 
```

Comment:
That is party my poor wording.

TokuDB uses a monotonically incrementing value as an internal file structure/descriptor id. There is a 'reserved' value of used to indicate 'null' or no value (~0). If you have a long running server with a lot of databases open/closes, this value can reach the storage capabilities (UINT32) and hit the reserved value instead of skipping the reserved value and reset to 0. This causes an assert because the calling code believes a file open operation failed when it in fact succeeded but just returned this reserved value instead of restarting at 0.